admin_guide/high_availability.adoc

```diff
@@ -302,44 +429,55 @@ endif::[]
 ifdef::openshift-origin[]
 ====
 ----
-$ oadm router ha-router-us-west --replicas=3 \
-    --selector="ha-router=geo-us-west" \
-    --labels="ha-router=geo-us-west" \
+$ oadm router ha-router-us-west --replicas=5 \
+    --selector="ha-svc-nodes=geo-us-west" \
+    --labels="ha-svc-nodes=geo-us-west" \
     --credentials="$KUBECONFIG" \
     --service-account=ipfailover
 ----
 ====
 endif::[]
 +
-[NOTE]
+Run the *geo-cache* service with a replica on each of the nodes. An example configuration
```

Comment:
@bmeng I clone openshift-docs and use asciibinder to format the text. The result looks correct to me.